### ISSUE TITLE
Model Performance

### ISSUE THREAD

**Issue by aiyb1314** [JunchenLiu77/Rip-NeRF#1]
Thank you for your outstanding work, when I follow the default config file to train the single scale dataset, I found that the training result is 0.3db different from the paper, please is there a specific super parameter config.

**Comment by JunchenLiu77** [comment]
Hi! I just update the default training iterations to 120k to align with the default setting as described in paper.

**Comment by aiyb1314** [comment]
I am referring to the difference in performance of the model when the number of iterations is 25k.

**Comment by aiyb1314** [comment]
According to the default parameter of config file, run the github code, the result in the paper is 34.92dB, but I actually run out to 34.5dB, is there any other super parameter setting.

**Comment by JunchenLiu77** [comment]
No, the default hyper params are aligned with the paper. But I made some minor modification compared with the original implementation, which may cause performance gap. I tested the released version on the ms-blender dataset and the performance is better than the result in paper.

**Comment by ballin0105** [comment]
> Thank you for your outstanding work, when I follow the default config file to train the single scale dataset, I found that the training result is 0.3db different from the paper, please is there a specific super parameter config.

Hi! Using the default config, I found the same gap in the single-scale dataset. Did you solve the problem?
